docs/developer-guide/checks.md

```diff
@@ -5,9 +5,15 @@ Here you can find how to create new checks for Prowler.
 **To create a check is required to have a Prowler provider service already created, so if the service is not present or the attribute you want to audit is not retrieved by the service, please refer to the [Service](./services.md) documentation.**
 
 ## Introduction
+
+The checks are the foundamental piece of Prowler. A check is a simply piece of code that ensure if something is configured whith best cybersecurity practices, or is not and could be a breach of a cloud infrastructure. Furthemore, some information about the audition that is being doing over that resource (*metadata*).
+
 To create a new check for a supported Prowler provider, you will need to create a folder with the check name inside the specific service for the selected provider.
 
-We are going to use the `ec2_ami_public` check form the `AWS` provider as an example. So the folder name will `prowler/providers/aws/services/ec2/ec2_ami_public` (following the format `prowler/providers/<provider>/services/<service>/<check_name>`), with the name of check following the pattern: `service_subservice/resource_action`.
+We are going to use the `ec2_ami_public` check form the `AWS` provider as an example. So the folder name will `prowler/providers/aws/services/ec2/ec2_ami_public` (following the format `prowler/providers/<provider>/services/<service>/<check_name>`), with the name of check following the pattern: `service_subservice_resource_action`.
+
+???+ note
+    A subservice is more specific part of a service that is gonna be audit. Sometime it could be the shortened name of the class attribute that is gonna be acceded in the check.
```

Comment:
```suggestion
    A subservice is an specific component of a service that is gonna be audited. Sometimes it could be the shortened name of the class attribute that is gonna be accessed in the check.
```